src/components/Date/SimpleDate.jsx

```diff
@@ -0,0 +1,211 @@
+import React, { useRef } from 'react';
+import PropTypes from 'prop-types';
+import _ from 'lodash';
+
+import Select from '../Select/Select';
+import NumberInput from '../NumberInput/NumberInput';
+import { months, days } from '../../helpers/options-for-select.js';
+
+const handleChange = (path, newValue, oldDate, onValueChange) => {
+  if (!onValueChange) return;
+
+  const date = {
+    ...oldDate,
+    [path]: newValue,
+  };
+
+  if (!date.month.value) {
+    date.day.value = '';
+  }
+
+  onValueChange(date);
+};
+
+// TODO: Refactor the date prop to be either a string or an object of strings
+
+/**
+ * A simple date picker with only display logic. For built-in validation, use the Date component.
+ */
+export const SimpleDate = ({
+  errorMessage,
+  date,
+  label,
+  name,
+  required,
+  ariaDescribedby,
+  minYear,
+  maxYear,
+  onValueChange,
+  onBlur,
+}) => {
+  const inputId = useRef(_.uniqueId('date-input-'));
```

Comment:
There was a recent PR (in `vets-website` I think) where we replaced all top-level lodash imports with their more specific counterparts, like:

```js
import { uniqueId } from 'lodash'
```

I think it would be good to do that here as well.